### PR TITLE
consolidated blockchain history log

### PR DESCRIPTION
**Pull request by howard-stearns**
Make wallet recent activity display match the new blockchain history that does atomic swaps.

(Requires https://github.com/highfidelity/data-web/pull/1151, so DNM until that is in production.)

### CODE REVIEW

**Comment by howard-stearns** [comment]
Test plan _by #discovery team testing in staging_:
1. With a _new_ (i.e., wiped) commerce backend (including blockchain), create a wallet and get seeded.
2. Buy something form the marketplace.  Buy something else.
3. Look at wallet -> recent history. Should show rows only for seeding, first purchase, and second purchase. Each row should show what was spent and what was received, with the number of HFC and certificates of each.

File: interface/src/commerce/Ledger.cpp [hunk 1]
```diff
@@ -129,6 +138,7 @@ void Ledger::historySuccess(QNetworkReply& reply) {
     // public key(s) are.  Let's keep it that way
     QByteArray response = reply.readAll();
     QJsonObject data = QJsonDocument::fromJson(response).object();
+    qInfo(commerce) << "history" << "response" << QJsonDocument(data).toJson(QJsonDocument::Compact);
```
Comment (howard-stearns) [inline interface/src/commerce/Ledger.cpp]: Our other commerce responses create info logs which helps in debugging, so now this one does to.

File: interface/src/commerce/Ledger.cpp [hunk 2]
```diff
@@ -141,32 +151,15 @@ void Ledger::historySuccess(QNetworkReply& reply) {
...
-            } else {
-                coloredQuantityAndAssetTitle = QString("<font color='#EA4C5F'>") + coloredQuantityAndAssetTitle + QString("</font>");
-            }
-        } else {
-            coloredQuantityAndAssetTitle = QString("\"<font color='#0093C5'><a href='") +
-                MARKETPLACE_ITEMS_BASE_URL +
-                valueObject["asset_id"].toString() +
-                QString("'>") +
-                coloredQuantityAndAssetTitle +
-                QString("</a></font>\"");
```
Comment (howard-stearns) [inline interface/src/commerce/Ledger.cpp]: I'd love to keep this link (eg., around the "n certificates" text), but I could not figure out how to get the asset id in  the new app/models/cache_chain.rb:89 of https://github.com/highfidelity/data-web/pull/1151/files, and we're out of time.

File: interface/src/commerce/Ledger.cpp [hunk 3]
```diff
@@ -141,32 +151,15 @@ void Ledger::historySuccess(QNetworkReply& reply) {
...
-        }
+        QString sent = amountString("sent", "EA4C5F", valueObject["sent_money"], valueObject["sent_certs"]);
+        QString received = amountString("received", "1FC6A6", valueObject["received_money"], valueObject["received_certs"]);
+
         // turns out on my machine, toLocalTime convert to some weird timezone, yet the
         // systemTimeZone is correct.  To avoid a strange bug with other's systems too, lets
         // be explicit
         QDateTime createdAt = QDateTime::fromSecsSinceEpoch(valueObject["created_at"].toInt(), Qt::UTC);
         QDateTime localCreatedAt = createdAt.toTimeZone(QTimeZone::systemTimeZone());
```
Comment (howard-stearns) [inline interface/src/commerce/Ledger.cpp]: I don't understand where this is used in the older code. Bug? I haven't (knowingly) changed it in the new code.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/5090/HighFidelity-Beta-PR11854-07d52306c4f677211a7c53519b1e32fab21a3f11.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5090/HighFidelity-Beta-PR11854-07d52306c4f677211a7c53519b1e32fab21a3f11.dmg

File: interface/src/commerce/Ledger.cpp (hunk 1 above)
Comment (davidkelly) [reply interface/src/commerce/Ledger.cpp]: 👍

**Pull request merged**